rust/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "keyvi"
-version = "0.4.6"
+version = "0.4.7"
```

Comment:
@vvucetic please see [release procedure](https://github.com/KeyviDev/keyvi/blob/master/doc/RELEASE_PROCESS.md) keyvi has.

on master branch it always has current dev/unreleased versions, so no need for this version bump here.